.gitignore

```diff
@@ -3,9 +3,13 @@ docs/_site
 docs/_dist
 docs/images/supporters
 mocha.js
+mocha.js.map
```

Comment:
why not ship this?

Reply:
We should totally ship the source map. I'll remove this line

Reply:
It's ignored in git like the other build artifacts. I did forget to add `mocha.js.map` to package.json `files` though. I'll do that

Reply:
Fixed in 6993cac8